tools/InstallationTests/Common.ps1

```diff
@@ -193,14 +193,28 @@ function Test-Setup
 #############################
 function Test-Cleanup
```

Comment:
Please ensure that these are the latest versions from the PowerShell share.  The files here are quite out of sync with the scripts we have been executing.